### PR TITLE
CB-18569 using new account for real ums tests, remove hardcoded accou…

### PR DESCRIPTION
**Pull request by horadla23**
…nt id from authz test

See detailed description in the commit message.

### CODE REVIEW

**Review by lnardai — APPROVED** [review]
Approved, but please make sure to check the cleanup utility.

File: integration-test/scripts/fetch-secrets.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 set -ex
 
-: ${SECRET_VERSION:="69f803e3ff3c42d79be1f87bb04341e9"}
+: ${SECRET_VERSION:="9fb177df00f0419aa0658ad18a041ee4"}
```
[the remark this answers is not in the document]
Comment (horadla23) [reply integration-test/scripts/fetch-secrets.sh:5]: is there any other job than this? http://ci-cloudbreak.eng.hortonworks.com/job/cloudbreak-ums-user-assignments-cleanup/
This was created by me, I am unaware of the one Kristof created, this one will be prepared with another PR: https://github.com/hortonworks/cloudbreak/pull/13455

**Comment by horadla23** [comment]
PR related UMS jobs still uses old account, but I can change it too

**Comment by horadla23** [comment]
retest this please

**Comment by horadla23** [comment]
retest this please

**Pull request merged**